shell/src/main/java/alluxio/cli/fsadmin/command/DoctorCommand.java

```diff
@@ -67,13 +67,14 @@ public int run(CommandLine cl) throws IOException {
     // Get the doctor category
     Command command = Command.ALL;
     if (args.length == 1) {
-      try {
-        command = Command.valueOf(args[0]);
-      } catch (IllegalArgumentException e) {
-        System.out.println(getUsage());
-        System.out.println(getDescription());
-        throw new IllegalArgumentException(String
-            .format("doctor category cannot be %s.", args[0]));
+      switch (args[0]) {
+        case "configuration":
+          command = Command.CONFIGURATION;
+          break;
+        default:
+          System.out.println(getUsage());
+          System.out.println(getDescription());
+          throw new InvalidArgumentException("doctor category is invalid.");
```

Comment:
optional: on error, perhaps we could list the valid options in the error message

Reply:
On error, we would print out the description. Description message contains all the valid category.